index.html

```diff
@@ -24,7 +24,7 @@
     </header>
 
```

Comment:
- [ ] Let's consider having the header/toolbar section match the Figma design. This is important as the desktop version doesn't require the menu button but to display the navigation links. 

Your header/toolbar section:
<img width="941" alt="Loom_RUTDTc0FTe" src="https://user-images.githubusercontent.com/81468675/149226991-213e3258-0e4a-49c7-9357-0090c187eaee.png">


Figma header/toolbar section:
<img width="445" alt="Loom_wuj6LZVl31" src="https://user-images.githubusercontent.com/81468675/149227516-639d1b3c-5c4c-4613-afc7-d49ff6aa9fca.png">